dist/index.js

```diff
@@ -669,13 +669,13 @@ class ClientConfig {
   baseDomain;
   clientId;
   clientSecret;
-  authUrl;
+  authDomain;
   constructor(config) {
     this.organisationId = config.organisationId;
     this.baseDomain = config.baseDomain || "https://app.ctrl-hub.com";
     this.clientId = config.clientId || "";
     this.clientSecret = config.clientSecret || "";
-    this.authUrl = config.authUrl || "";
+    this.authDomain = config.authDomain || "";
```

Comment:
this should have a default `https://auth.ctrl-hub.com`

Reply:
Fixed - I'm thinking about getting rid of the ClientConfig itself, I don't think it adds anything better than simply having an interface with the desired fields & optional fields - but can come back to that